### PR TITLE
Populate autoscale column for nodepools

### PR DESCRIPTION
**Pull request by o-farag**
Signed-off-by: Omar Farag <ofarag@redhat.com>

https://issues.redhat.com/browse/ACM-2663

Autoscaling column shows `False` if autoscaling is disabled; the Min and Max size of the pool otherwise.
image
image

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx [hunk 1]
```diff
@@ -86,17 +86,20 @@ const NodePoolsTable = ({ nodePools, clusterImages }: NodePoolsTableProps): JSX.
...
+            }
 
-        const min = autoscaling.min || 1
-        const max = autoscaling.max || 1
+            const min = autoscaling.min || 1
+            const max = autoscaling.max || 1
 
-        return `Min ${min} Max ${max}`
-    }, [])
+            return `Min ${min} Max ${max}`
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx:99]: This string needs to be externalized for translation.

Comment (o-farag) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx]: Done. Not sure if that was the best way to do it, let me know if there's a better way.

Comment (KevinFCormier) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx]: Probably better to use interpolation, like `t('Min {{min}} Max {{max}}', {min, max})`. When you commit, it should automatically add to translations.json and remove the old ones.

Comment (o-farag) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx]: Thanks, I tried something similar but couldn't get it to work

**Review by chenz4027 — APPROVED** [review]
/lgtm thanks Omar :)
/hold for @KevinFCormier to take another look

**Pull request merged**